conda/recipes/librmm/meta.yaml

```
@@ -85,25 +79,15 @@ outputs:
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
         - {{ compiler('cuda') }}
-        {% if cuda_major != "11" %}
-        - cuda-cudart-dev
-        {% endif %}
+        - {{ "cuda-cudart-dev" if cuda_major == "12" else "cuda-version" }}
```

`conda-recipe-manager` doesn't like `!=` as a comparison operator

---

Maybe we could do `not` and `==`s?

```suggestion
        - {{ "cudatoolkit" if not (cuda_major == "12") else "cuda-version" }}
```

---

Yep, that works!

---

We want to write this in a way that only mentions CUDA 11, so that the condition can be trivially deleted when adding future major version support.

---

Yeah that was the idea. I just goofed on the syntax. Took another go below

https://github.com/rapidsai/rmm/pull/1796#discussion_r1931379512